apps/03TicTacToe.js

```diff
@@ -14,6 +14,10 @@ var board = [
 
 var playerTurn = 'X';
 
+function togglePlayerTurn() {
+  playerTurn = (playerTurn === 'X') ? 'X' : 'O';
```

Comment:
Still have slight problem with the ternary logic. Write out this logic with a normal IF statement if that will help you see the issue.